R/button.R

```diff
@@ -158,7 +171,7 @@ counterbutton <- function(name, label = "", icon = NULL, value = 0, color = "",
 
 #' @rdname counterbutton
 #' @export
-counterButton <- function(name, label = "", icon = NULL, value = 0, color = "", size = "",
-              big_mark = " ") {
-  counterbutton(name, label, icon, value, color, size, big_mark)
+counterButton <- function(inputId, label = "", icon = NULL, value = 0,
```

Comment:
Should we even define this function? There is no counterButton in shiny, so we should stay with counter_button only.

Reply:
not married to it, but since it's so similar to action button, I thought it would be nice to support the user with analogous function.

Reply:
Hmm, if we want to use our current naming rules I'd still insist to remove it.